_targets.R

```diff
@@ -0,0 +1,42 @@
+library(targets)
+source("code.R")
+tar_option_set(packages = c("tidyverse", "stringr", "sbtools", "whisker"))
+
+list(
+  # Get the data from ScienceBase
+  tar_target(
+    model_RMSEs_csv,
+    download_data(out_filepath = "model_RMSEs.csv"),
```

Comment:
Edit all of your filepaths to match  your repo's organization - so things that are "fetch" targets should end up in `1_fetch/out`, etc.